back-end/sports_platform/sports_platform_api/views/user_views.py

```diff
@@ -50,8 +50,16 @@ def get_user(request, user_id):
             return Response(status=200)
         except Exception:
             return Response(data={'message': 'An error occured, please try again later.'}, status=500)
-             
-        
+    elif request.method == 'DELETE':
+        if not request.user.is_authenticated:
+            return Response({"message": "User not logged in."},
+                        status=401)
```

Comment:
I think here we should check if the user is trying to delete their own account.

Reply:
great point, thanks!